api/print.php

```diff
@@ -51,9 +55,9 @@
         $source = imagecreatefromjpeg($filename_source);
         $code = imagecreatefrompng($filename_codes);
 
-        if ($config['print_frame'] == true) {
+        if ($config['print_frame'] == true && !($config['take_frame'])) {
```

Comment:
Not related to this pr, but we can remove ` == true`.

Reply:
We can do that here

Reply:
done